package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@duneanalytics/hooks",
-  "version": "1.0.8",
+  "version": "2.0.0",
```

Comment:
Are we sure we want to bump to 2? Another approach is keep the useTransactions and pass an object config for vm: [evm|svm] with evm as the default.

Reply:
I've included some breaking changes, hence the major bump, but I could revert those. There are some inconsistencies in the API I'd like us to fix, but it's not necessary.